### ISSUE TITLE
Decreased urinary creatinine level

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4858]
**Preferred term label:**
Decreased urinary creatinine level

**Synonyms**
 Low Urinary Creatinine Excretion

**Definition (free text, please give PubMed ID)**
A decrease of creatinine concentration compared to the normal 

**Parent term (use hpo.jax.org/app)**
Abnormality of urine homeostasis HP:0003110

(Sibling term: creatininuria)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0001-9969-8610

**Comment by pnrobinson** [comment]
I do not think this is a valid phenotype. the urinary creatine concentration depends on the urine specific gravity and urine creatine levels are used basically to normalize other metabolite concentrations. If I am wrong please provide more information and a pubmed reference!

**Comment by pnrobinson** [comment]
I think this is not an abnormal phenotype
